### ISSUE TITLE
train

### ISSUE THREAD

**Issue by suxingzhao** [ibaiGorordo/CREStereo-Pytorch#10]
Hi, I noticed that you have a train.py script in your repo, but you didn't mention anything about training directly with your repo. Is it fully implemented? If not, could you kindly share what is missing and maybe I can try to add things to make it work. Thanks!

**Comment by ibaiGorordo** [comment]
I haven't tried, but it should work. Just follow the steps in the original repository for preparing the data and it should work.

**Comment by suxingzhao** [comment]
Thank you. I have tried my own data set, but the loss has not decreased. I suspect there is a problem with back propagation during training

**Comment by ibaiGorordo** [comment]
Not sure, another option is to train using the original repository and use the convert weights script in this repository to convert it to pytorch for.inference.

**Comment by suxingzhao** [comment]
https://github.com/megvii-research/CREStereo/issues/32
This problem occurs when I use the original repository for training. Shape is not correct

**Comment by ibaiGorordo** [comment]
The issue is probably because you changed the input width. It should be divisible by 8 (the height is fine)

**Comment by suxingzhao** [comment]
It seems that this is not the problem，My input is 640 * 480

**Comment by ynma-hanvo** [comment]
> It seems that this is not the problem，My input is 640 * 480

@suxingzhao  hi,  did you succeed with training with this pytorch implmentation.
